packages/@okta/vuepress-site/docs/guides/token-inline-hook/test/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Test the Token Inline Hook
+---
+
+The Token Inline Hook is now ready for testing. You now have the following applications configured:
+
+- The Okta-Hosted-Login sample application (`samples-nodejs-express-4`) is ready to authenticate users from your Okta org.
+- The external service (Glitch.com project) is ready with code to receive and respond to an Okta Token Inline Hook call.
+- The Okta org is setup to call the external service when a Token Inline Hook is triggered by a user sign-in from the Okta-Hosted-Login sample application, and ready to receive a response.
+
+### Test your hook
+
+1. Navigate to your sample application project folder (`samples-nodejs-express-4`).
+
+2. Start your Okta-Hosted-Login server (`npm run okta-hosted-login-server`).
+
+3. Navigate to your sample application (`http://localhost:8080`).
+
+4. Navigate to your Glitch.com project and open the Console Logs window (**Tools** > **Logs**).
+
+5. Return to your sample application and sign in with an Okta user who is NOT in the Patients data store.
+
+    - The user should sign in as normal; only the user name should appear in the Glitch log window.
+    - If you extended the sample application, click on `Profile` in the left-hand navigation pane. Only the the user info claims are included in the table.
```

Comment:
The left navigation appears as "My Profile" instead of "Profile" in my sample app:
<img width="828" alt="Screen Shot 2021-02-26 at 4 46 41 PM" src="https://user-images.githubusercontent.com/67654866/109369718-45e1a800-7852-11eb-9e98-f8d1202218c6.png">

Reply:
Updated. Thanks.